scss/components/_date-grid.scss

```diff
@@ -0,0 +1,80 @@
+// Date Range Calendar Grid Date Picker
+//
+// Filter interaction for filter date fields
+//
+
+.date-range-grid {
+  @include clearfix();
+  font-family: $sans-serif;
+  padding: u(2rem 0);
+  display: none;
+
+  &:not(.is-invalid) {
+
+    &.pick-min li:hover div {
+      border-top-left-radius: u(1.5rem);
+      border-bottom-left-radius: u(1.5rem);
+      background-color: $aqua;
+      color: $base !important;
+    }
+
+    &.pick-max li:hover div {
+      border-top-right-radius: u(1.5rem);
+      border-bottom-right-radius: u(1.5rem);
+      background-color: $aqua;
+      color: $base !important;
+    }
+
+    &.pick-min,
+    &.pick-max {
+      li {
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+.date-range-year {
+  line-height: u(3rem);
+  float: left;
+  clear: both;
+}
+
+.date-range-months {
+  width: 220px;
+  float: right;
+
+  li {
+    width: 35px;
+    color: $base !important;
+    font-size: u(1.2rem);
+    text-align: center;
+    padding: u(0 0 .2rem .1rem);
+    float: left;
+
+    div {
+      background-color: $gray;
+      padding: u(.75rem .5rem);
+    }
+
+    &.month--begin div {
+      border-top-left-radius: u(1.5rem);
+      border-bottom-left-radius: u(1.5rem);
+    }
+
+    &.month--end div {
+      border-top-right-radius: u(1.5rem);
+      border-bottom-right-radius: u(1.5rem);
+    }
+
+    &.selected div {
+      background-color: $gray-dark;
+      color: $inverse !important;
+    }
+
+    &.is-active div {
+      background-color: $aqua;
+      color: $base !important;
```

Comment:
Yes, thank you for the reminders. It was necessary when I was trying to change the color of just the `li` elements but since now the colors are set in a child `<div>` that has a specificity high enough not to need the importants.